cms/pages/templates/pages/base_page.html

```diff
@@ -56,9 +56,8 @@ <h1>{{ self.title }}</h1>
         </div>
         <div class="nhsuk-grid-column-one-third">
             {% if children %}
-            <h2>In this section</h2>
-            <nav class="nhsuk-contents-list" role="navigation" aria-label="Pages in this guide">
-                <h2 class="nhsuk-u-visually-hidden">Contents</h2>
+            <nav class="nhsuk-contents-list" role="navigation" aria-label="Pages in this section" aria-labbeledby="sideNav">
```

Comment:
I'm not sure if aria-label is needed anymore, if I added aria-lebbeled by with a reference to the Heading that describes and is now visible 🤷🏻‍♀️